README.md

```diff
@@ -6,4 +6,33 @@ Welcome to the GitHub Workflows repository of AI-CFIA! This repository contains
 
 This repository is dedicated to providing automated solutions and workflows for various operations within our GitHub projects. It contains a collection of reusable workflows for the CI/CD pipelines, testing, and other automated processes within GitHub Actions.
 
-Find out how to use these workflows [here](./github-workflows-documentation.md)
+Each workflow has an associated documentation file that explains its purpose, usage, and any required secrets or inputs. You can find these files in the same directory as their corresponding workflow file.
+
+## Workflow Documentation
+
+Here are the available workflows and their associated documentation:
+
+- [Reusable Docker Container Build](./workflow-build-container.md)
```

Comment:
you don't need ```./``` prefix to reference files in the same directory as the current document

Reply:
they are in the .github/workflows folder as you said you wanted the .md with the .yml but yeah just realized my hyperlinks were wrong I will change the markdown link testing so that it runs on this repository as well